org/ystia/yorc/yorc/pub/types.yaml

```diff
@@ -169,6 +169,28 @@ node_types:
       config:
         type: org.ystia.yorc.pub.capabilities.YorcConfigContainer
 
+  org.ystia.yorc.pub.TerraformConfig:
+    derived_from: tosca.nodes.Root
+    description: Terraform configuration for Yorc
+    abstract: true
+    properties:
+      plugins_dir:
+        description: "The directory where are located Terraform plugins"
+        type: string
+        required: true
+        default: /var/terraform/plugins
+      plugins_download_urls:
+        description: "List of Terraform plugins download urls"
+        type: list
+        required: true
+        entry_schema:
+          type: string
+    requirements:
+    - host:
+        capability: org.ystia.yorc.pub.capabilities.YorcConfigContainer
```

Comment:
Is it the good way of doing this ? Here you say that the TerraformConfig is hosted on Yorc. It's great, it works. But, I thought that it should be hosted on the TerraformRuntime as it is terraform that you are modifying.

The problem is, that if you change this to be hosted on TerraformRuntime, you cannot update the yorc config file to put the terraform plugins dir... We should discuss about it.

Reply:
Yes, furthermore, there is no straight relationship btw TerraformRuntime and the TF plugins.